tsconfig.json

```diff
@@ -24,6 +25,7 @@
       "@defichain/jellyfish-address/*": ["jellyfish-address/src/*"],
       "@defichain/jellyfish-api-core/*": ["jellyfish-api-core/src/*"],
       "@defichain/jellyfish-api-jsonrpc/*": ["jellyfish-api-jsonrpc/src/*"],
+      "@defichain/jellyfish-block/*": ["jellyfish-block/src/*"],
```

Comment:
FYI, the `/*` will be needed to be removed
https://github.com/DeFiCh/jellyfish/pull/658